### ISSUE TITLE
twisted/python/test/test_util.py emits DeprecationWarning

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6324]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)| @tomprince reported|
|-|-|
|Trac ID|trac#6324|
|Type|defect|
|Created|2013-02-24 08:31:15Z|
|Branch|https://github.com/twisted/twisted/tree/python.test_util-emits-DeprecationWarning-6324|

<details><summary>Searchable metadata</summary>

```
trac-id__6324 6324
type__defect defect
reporter__tom_prince tom.prince
priority__normal normal
milestone__ 
branch__branches_python_test_util_emits_DeprecationWarning_6324 branches/python.test_util-emits-DeprecationWarning-6324
branch_author__tomprince tomprince
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy easy
time__1361694675000000 1361694675000000
changetime__1362338182000000 1362338182000000
version__None None
owner__tom_prince tom.prince

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_1"></a>|@tomprince commented|
|-|-|

(In [37223]) Branching to python.test_util-emits-DeprecationWarning-6324.

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_3"></a>|@wallrj commented|
|-|-|

 * Started [a build](http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/python.test_util-emits-DeprecationWarning-6324)
 * See ticket:6323#comment:4

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_4"></a>|@wallrj set owner to @tomprince|
|-|-|

Code Review:

Looks good.

 * Change looks fine
 * Build results are mostly fine.
 * The getPluginDir deprecation warning no longer shows up in python 2.6 builds
   * https://buildbot.twistedmatrix.com/builders/lucid32-py2.6-select/builds/2538/steps/trial/logs/warnings
   * vs https://buildbot.twistedmatrix.com/builders/lucid32-py2.6-select/builds/2540/steps/trial/logs/warnings

 1. One new twistedchecker warning which may as well be fixed
    ```
    ************* Module twisted.python.test.test_util
    W9013:1045,0: Expected 3 blank lines, found 4
    ```

+1 Fix the twistedchecker warning and merge.

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_5"></a>|@tomprince set status to `closed`|
|-|-|

(In [37373]) Merge python.test_util-emits-DeprecationWarning-6324: Filter expected DeprecationWarnings from twisted.python.test.test_util.

Author: tom.prince
Reviewers: rwall
Fixes: [#6324](https://github.com/twisted/twisted/issues/6324)

twisted.python.util.getPluginDir is deprecated, but need to be tested.
Mask the deprecation warnings until they can be removed.
